packages/@okta/i18n/src/properties/login.properties

```diff
@@ -819,6 +819,11 @@ oie.verify.google_authenticator.otp.title = Verify with Google Authenticator
 oie.verify.google_authenticator.otp.description = Enter the temporary code generated in your Google Authenticator app
 oie.google_authenticator.otp.enterCodeText = Enter code
 
+## Number Challenge
+oie.numberchallenge.warning = Haven't received a push notification yet? Try opening the Okta Verify app on your device, or <$1>resend the push notification</$1>.
+## argument {0} in oie.numberchallenge.instruction is a number
```

Comment:
Please refer to the i18n-smartling format: ## {0} refers to xxx

Reply:
Thanks, can you point me to a wiki ?

Reply:
Updating this to `## {0} refers to a number which is the correct answer for the number challenge`